clientv3/integration/cluster_test.go

```diff
@@ -62,6 +63,39 @@ func TestMemberAdd(t *testing.T) {
 	}
 }
 
+func TestMemberRemoveWithAutoSync(t *testing.T) {
```

Comment:
This is only testing that the client is getting the member list when the important part is really client connectivity. A good test of functionality would be something like:
1) boot a cluster with one member
2) attach an autosync client
3) add two more members
4) wait for sync to grab the new members
5) stop the original member
6) use the client to access the cluster (the rpc will only go through if the sync worked)